src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double countArea(Figure figure);
```

Comment:
we have direct access to the figures fields as this method will be included in the Figure class
```suggestion
    double countArea();
```